spec/support/schemas/claims_api/v2/veterans/claims/claims.json

```diff
@@ -52,6 +52,16 @@
         "nullable": true,
         "description": "If true, indicates a decision has been requested and/or a Waiver 5103 has been submitted",
         "example": "false"
+      },
+      "endProductCode": {
+        "type": "string",
+        "description": "End product code of claim"
+      },
+      "closeDate": {
+        "type": ["string", "null"],
+        "format": "date",
+        "description": "Date claim was closed",
+        "example": "2019-09-04"
```

Comment:
I've been trying to keep all the attribute names in these JSON schemas in alphabetical order.  (just to make it quicker/easier to identify things).  Can we get these new attributes alphabetized as well?